README.md

```diff
@@ -18,6 +18,13 @@ end
 
 ExFaiss will download, build, and cache Faiss on the first compilation. You must have CMake installed in order to build Faiss.
 
+### MacOS Compilation
+For some reason ExFaiss doesn't want to build using the Apple LLVM so to make this work you will first need to install LLVM via [Homebrew](https://brew.sh/):
```

Comment:
Also, fwiw, I am using Apple LLVM (macOS 12.x).

Reply:
For some reason on (macOS 13.3 (22E252)) it does not.